L/Librsvg/build_tarballs.jl

```diff
@@ -75,7 +105,7 @@ dependencies = [
     HostBuildDependency("gdk_pixbuf_jll"),
     BuildDependency("Xorg_xorgproto_jll"),
     Dependency("gdk_pixbuf_jll"),
-    Dependency("Pango_jll"; compat="1.47.0"),
+    Dependency("Pango_jll"),
```

Comment:
Don't remove compat bounds.  If you need to, bump it.

Reply:
The recipe is an upgrade. If I remember correctly, I must set it to `compat="1.50.0"` for this version. It worked without bounds, so I proceeded that way.

Reply:
Ok.